### DISCUSSION TITLE
Discogs integration

### DISCUSSION THREAD

**Discussion by kristojorg** [jpochyla/psst#93] [Ideas]
Hi there,

I love that you made this project! Thank you! I have personally always despised how music clients allow you to browse music. Discogs, on the other hand, is such an amazing little graph of music information that is pretty underutilized. I would love to be able to eg. see the producers on an album, click their names, see other things they produced, easily listen to those. Follow record labels, maybe, etc.

Even more extravagant would be other ways to browse music, like by sub-genre or style, or by date and location. For example, I hear some song I love and want to hear other songs from that region and time. 

Long story short, I think Spotify does an abysmal job with music discovery (relying too heavily on algorithms), and the possibility of browsing in new ways, similar to whats possible on discogs, would be exciting. I don't know if a discogs integration is possible, but thought I would bring this topic up just to start the discussion : ). 

Cheers!

Kristo

**Comment by jdille** [comment]
Huge fan of this integration idea and I agree, would love to see something like this.

**Comment by secondlnl** [comment]
As someone not at all familiar with Discogs (with a quick search it just seems like curated lists, a marketplace and wish lists) how would this integration idea work in practice, at the same time wouldn't it add clutter. This would also need to be something further down the pipeline. (Just thoughts)
